docs/content/concepts/assets/asset-jobs.mdx

```diff
@@ -0,0 +1,194 @@
+
+### Hardcoded configuration
+
+You can supply a <PyObject object="RunConfig"/> object or raw config dictionary. The supplied config will be used to configure the job whenever the job is launched. It will show up in the UI Launchpad and can be overridden.
+
+```python file=/concepts/ops_jobs_graphs/jobs_with_default_config.py
+from dagster import Config, RunConfig, job, op
+
+
+class DoSomethingConfig(Config):
+    config_param: str
+
+
+@op
+def do_something(context, config: DoSomethingConfig):
+    context.log.info("config_param: " + config.config_param)
+
+
+default_config = RunConfig(
+    ops={"do_something": DoSomethingConfig(config_param="stuff")}
+)
+
+
+@job(config=default_config)
+def do_it_all_with_default_config():
+    do_something()
+
+
+if __name__ == "__main__":
+    # Will log "config_param: stuff"
+    do_it_all_with_default_config.execute_in_process()
+```
+
+### Partitioned configuration
+
+For op-based jobs, you can supply a <PyObject object="PartitionedConfig" /> to create a partitioned job. This defines a discrete set of partitions along with a function for generating config for a partition. Job runs can be configured by selecting a partition.
+
+Refer to the [Partitions documentation](/concepts/partitions-schedules-sensors/partitions) for more info and examples.
+
+### Config mapping
+
+You can supply a <PyObject object="ConfigMapping" />. This allows you to expose a narrower config interface to your job. Instead of needing to configure every op and resource individually when launching the job, you can supply a smaller number of values to the outer config, and the <PyObject object="ConfigMapping" /> can translate it into config for all the job's ops and resources.
+
+```python file=/concepts/ops_jobs_graphs/jobs_with_config_mapping.py
+from dagster import Config, RunConfig, config_mapping, job, op
+
+
+class DoSomethingConfig(Config):
+    config_param: str
+
+
+@op
+def do_something(context, config: DoSomethingConfig) -> None:
+    context.log.info("config_param: " + config.config_param)
+
+
+class SimplifiedConfig(Config):
+    simplified_param: str
+
+
+@config_mapping
+def simplified_config(val: SimplifiedConfig) -> RunConfig:
+    return RunConfig(
+        ops={"do_something": DoSomethingConfig(config_param=val.simplified_param)}
+    )
+
+
+@job(config=simplified_config)
+def do_it_all_with_simplified_config():
+    do_something()
+
+
+if __name__ == "__main__":
+    # Will log "config_param: stuff"
+    do_it_all_with_simplified_config.execute_in_process(
+        run_config={"simplified_param": "stuff"}
+    )
+``` -->
+
+---
+
+## Making jobs available to Dagster tools
+
+You make jobs available to the UI, GraphQL, and the command line by including them in <PyObject object="Definitions"/> object at the top-level of Python module or file. The tool loads that module as a code location. If you include schedules or sensors, the code location will automatically include jobs that those schedules or sensors target.
```

Comment:
```suggestion
You make jobs available to the Dagster UI, GraphQL, and the command line by including them in <PyObject object="Definitions"/> object at the top level of a Python module or file. The tool loads that module as a code location. If you include schedules or sensors, the code location will automatically include jobs that those schedules or sensors target.
```

Reply:
Consider adding a link to the Code location docs in here somewhere: `/concepts/code-locations`